BooruSharp.UnitTests/BooruTests.cs

```diff
@@ -297,8 +297,8 @@ public async Task GetLastPostsWithTagsAsync(Type t, string tag = "hibiki_(kantai
             Assert.NotEqual(results[0].ID, results[1].ID);
             foreach (var elem in results)
             {
-                Assert.Contains(elem.Tags, t => t.Contains(tag));
-                Assert.Contains(elem.Tags, t => t.Contains(tag2));
+                Assert.Contains(elem.Tags, x => x.Contains(tag));
+                Assert.Contains(elem.Tags, x => x.Contains(tag2));
```

Comment:
What is the reason for this change?

Reply:
Naming it t hides the method parameter t. Though now I noticed it's still able to compile it as t so the change is actually pointless.

Reply:
Ah yup I didn't notice on the changes that the parameter was named the same way, nice catch